util/util.go

```diff
@@ -87,7 +88,7 @@ type ThingSpec struct {
 
 // Patch will patch a particular resource
 func Patch(restclient *rest.RESTClient, path string, value string, resource string, name string, namespace string) error {
-
+	log.Info("IN PATCH..... IS IT BROKEN?!?!?!?")
```

Comment:
Assuming this was a development log message :)

Reply:
lol, yeah. And I **thought** I caught that, but I guess the push didn't happen ;) Fixed now, thanks.